update and rebuild merge conflict

ryanpeek opened the pull request:
Looks like there was a merge conflict at some point on this page, and there were still tags for the HEAD and duplicated sections. I made one tiny change and pushed this up, so hopefully on the rebuild it will eliminate this issue. Thanks!

```
<<<<<<< HEAD
```

MarthaWohlfeil commented:
Looks like the merge conflict still isn't fixed. I'll look into it today

ryanpeek commented:
Did you pull and run into a conflict? It must be local because I don’t have any problem. I was able to build the site as well, so if you want I can build and push to master to update.  You may want to try deleting your local contact.rmd file and then pull so it doesn’t conflict. 

Sent from my iPhone

> On Feb 14, 2019, at 9:07 AM, Martha Wohlfeil <notifications@github.com> wrote:
> 
> Looks like the merge conflict still isn't fixed. I'll look into it today
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or mute the thread.

MarthaWohlfeil commented:
It must be local. My local website has been messed up for a week or two
now, I just haven't had the change to fully fix it. I'll try to today and
then pull and build the site and make sure everything looks ok.

Martha

On Thu, Feb 14, 2019 at 9:15 AM Ryan Peek <notifications@github.com> wrote:

> Did you pull and run into a conflict? It must be local because I don’t
> have any problem. I was able to build the site as well, so if you want I
> can build and push to master to update. You may want to try deleting your
> local contact.rmd file and then pull so it doesn’t conflict.
>
> Sent from my iPhone
>
> > On Feb 14, 2019, at 9:07 AM, Martha Wohlfeil <notifications@github.com>
> wrote:
> >
> > Looks like the merge conflict still isn't fixed. I'll look into it today
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/gge-ucd/R-DAVIS/pull/79#issuecomment-463711597>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AXzufN_ptpT99irdv49hziEkl7ut60taks5vNZnEgaJpZM4arkgo>
> .
>


-- 
Martha Wohlfeil

Graduate Group in Ecology
Avian Ecology Lab, PhD Candidate
Wickson Hall 3017
University of California, Davis
marthawohlfeil.github.io

ryanpeek commented:
Hi y'all...I went ahead and rebuilt the site and pushed to master. It's
updated with the fix on the contacts page. I am using the R-DAVIS site as a
reference/example for some job/fellowship stuff so wanted to get it
updated. Just an FYI you'll want to pull first (as always). Let me know if
you still have local conflicts Martha, I can help troubleshoot on Tuesday
if you need.

One note, I'm getting a warning (but still successful build) which relates
to the drop down answer/menu (I think it's on a few of the pages)...anyone
else get this when building site? And any suggestions for fixing?

Warning messages:
1: In get_engine(options$engine) :
  Unknown language engine 'text_answer' (must be registered via
knit_engines$set()).


Thanks!
Adios,
R

On Thu, Feb 14, 2019 at 9:42 AM Martha Wohlfeil <notifications@github.com>
wrote:

> It must be local. My local website has been messed up for a week or two
> now, I just haven't had the change to fully fix it. I'll try to today and
> then pull and build the site and make sure everything looks ok.
>
> Martha
>
> On Thu, Feb 14, 2019 at 9:15 AM Ryan Peek <notifications@github.com>
> wrote:
>
> > Did you pull and run into a conflict? It must be local because I don’t
> > have any problem. I was able to build the site as well, so if you want I
> > can build and push to master to update. You may want to try deleting your
> > local contact.rmd file and then pull so it doesn’t conflict.
> >
> > Sent from my iPhone
> >
> > > On Feb 14, 2019, at 9:07 AM, Martha Wohlfeil <notifications@github.com
> >
> > wrote:
> > >
> > > Looks like the merge conflict still isn't fixed. I'll look into it
> today
> > >
> > > —
> > > You are receiving this because you modified the open/close state.
> > > Reply to this email directly, view it on GitHub, or mute the thread.
> >
> > —
> > You are receiving this because your review was requested.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/gge-ucd/R-DAVIS/pull/79#issuecomment-463711597>, or
> mute
> > the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AXzufN_ptpT99irdv49hziEkl7ut60taks5vNZnEgaJpZM4arkgo
> >
> > .
> >
>
>
> --
> Martha Wohlfeil
>
> Graduate Group in Ecology
> Avian Ecology Lab, PhD Candidate
> Wickson Hall 3017
> University of California, Davis
> marthawohlfeil.github.io
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/gge-ucd/R-DAVIS/pull/79#issuecomment-463720898>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AFl9sCGh-xBOdv_FW-95zMTbQ10whuU1ks5vNZ_zgaJpZM4arkgo>
> .
>


-- 




*"When we try to pick out anything by itself, we find it hitched to
everything else in the universe."John Muir (My First Summer in the Sierra,
1911)*
*-----------------------------------------------------*
Ryan Peek, PhD
Ecologist
Center for Watershed Sciences, UC Davis
*ryanpeek.github.io <http://ryanpeek.github.io>*
*@riverpeek*
530.754.5351


*-----------------------------------------------------*
